Resources/Prototypes/_Mono/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -0,0 +1,35 @@
+        Blunt: 0.2
+        Slash: 0.2
+        Piercing: 0.2
+        Heat: 0.2
+        Cold: 0.35
+        Radiation: 0.05
+        Caustic: 0.2
+  - type: ClothingSpeedModifier
+    walkModifier: 0.8
+    sprintModifier: 0.8
+  - type: HeldSpeedModifier
+  - type: ToggleableClothing # Goobstation - Modsuits change
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)